src/module/module-collection.js

```diff
@@ -53,6 +53,14 @@ export default class ModuleCollection {
 
     parent.removeChild(key)
   }
+
+  isRegistered (path) {
+    const parent = this.get(path.slice(0, -1))
```

Comment:
I'm not sure why this method is semantically related with `mkdir`. They are used on different platform (Web/CLI) and also are different kind of task (`hasModule` *checks* a module while `mkdir` *creates* a directory).

If the parent module does not exist, it's clear that its child module also does not exist. So why `hasModule` need to throw in that case? 🤔